### ISSUE TITLE
Wifi

### ISSUE THREAD

**Issue by Majkwin** [Kaijun/Acer-V5-573G-Hackintosh#4]
Is there any possibility to enable wifi on this laptop under osx? Have you changed internal wifi card to do that?

**Comment by wilsoxu** [comment]
Hi all, 

Do any of you have the problem of your wifi not being able to turn on after you sleep and wakeup the laptop?

**Comment by Kaijun** [comment]
@wilsoxu 
right, sometimes it happens actually, but very rarely i would say. I just met this problem only twice more or less after i'm using OSX for more than one year.

But i found a solution: Boot into Windows and toggle your wifi several times. Then reboot to OSX and generally it will fix.

Have a try and let me know if it works indeed.
